### PR TITLE
Publish fr/lecons/preserver-logiciels-recherche

### LINKED ISSUE #3406: Preparing to publish a new FR lesson: preserver-logiciels-recherche
I am opening this Issue to prepare a new FR lesson for publication.

### PR DESCRIPTION
**Pull request by anisa-hawes**
Preparing files for publication.

Closes #3406 

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Add the appropriate "Label"
- [x] If this PR closes an Issue, add the phrase `Closes #ISSUENUMBER` to your summary above
- [x] Ensure the status checks pass: if you have difficulty fixing build errors, please contact our Publishing Manager @anisa-hawes 
- [ ] Check the Netlify Preview: navigate to netlify/ph-preview/deploy-preview and click 'details' (at right)
- [ ] Assign at least one individual or team to "Reviewers"
  - [ ] if the text needs to be translated, please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines), then assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing editor in your PR.

### CODE REVIEW

**Comment by anisa-hawes** [comment]
Note:

This build is failing due to 404 errors returned by the following pages of the Software Heritage website:

https://archive.softwareheritage.org/browse/directory/
https://archive.softwareheritage.org/browse/release/
https://archive.softwareheritage.org/browse/revision/
https://archive.softwareheritage.org/browse/snapshot/

I've asked the authors to help us to identify alternative links. _We also need to clarify if the website structure has changed recently, and if anything else in the lesson need to be updated to reflect these changes._

**Comment by anisa-hawes** [comment]
Bonjour Marie @marie-flesch.

I’ve staged preserver-logiciels-recherche for publication. You can [read it through in Preview](https://deploy-preview-3407--ph-preview.netlify.app/) and let me know if you note anything which you'd like to adjust.

With thanks to Frédéric, the problem with the links is solved! - these were intended to explain how the first part of a URL will be structured, rather than to be functional links so we have reformatted them to reflect this ([Option 5 Cas d’étude : explorer Software Heritage](https://deploy-preview-3407--ph-preview.netlify.app/fr/lecons/preserver-logiciels-recherche#cas-d%C3%A9tude-explorer-software-heritage).

**Note**: We are still waiting for one of the authors to confirm their bio.

Thank you!

File: fr/lecons/preserver-logiciels-recherche.md [hunk 1]
```diff
@@ -0,0 +1,474 @@
...
+- Chiara Mainardi
+- Magaye Sall
+- Martin Amouzou
+editors:
+- Daphné Mathelier
+review-ticket: https://github.com/programminghistorian/ph-submissions/issues/616
+difficulty: 1 
+activity: sustaining 
+topics: [data-management]
+abstract: Cette leçon présente des modalités d'archivage adaptées aux spécificités du logiciel et à différents profils d'utilisateurs. Les bonnes pratiques pour citer les logiciels, ainsi que des méthodes pour explorer efficacement les fonds de Software Heritage, seront abordées.
```
Comment (marie-flesch) [inline fr/lecons/preserver-logiciels-recherche.md]: Just a minor change to make this gender-inclusive. Change "utilisateurs" for "utilisateur·ices" at the end of first sentence, like so: 
"Cette leçon présente des modalités d’archivage adaptées aux spécificités du logiciel et à différents profils d’utilisateur·ices."

Comment (anisa-hawes) [reply fr/lecons/preserver-logiciels-recherche.md]: Merci @marie-flesch!

I have made that change.

Also just worked out how to type the _point médian_ with option + shift + 9 so we are happy ☺️

**Review by marie-flesch — APPROVED** [review]
All good for publication!

**Pull request merged**